dev/source/docs/how-the-team-works.rst

```diff
@@ -18,4 +18,4 @@ These pages are less about the software directly and more about the procedures a
     Roadmap <roadmap>
     Top Contributors <common-team>
     Voting <how-the-team-works-voting>
-    Weekly meetings on Mumble Server <ardupilot-mumble-server>
```

Comment:
must move to archived topics document to keep in TOC

Reply:
Done.